scripts/widgets/mz-search-pagination-dd.js

```diff
@@ -0,0 +1,168 @@
+﻿
+define(["jquery", "underscore"], function ($, _) {
```

Comment:
This is defined as a jQuery plugin/function instead of as a module in Core Theme. It's not following the AMD module pattern everything else is. Ideally this would be a module that returns a `mozuDropdown` or `createDropdown` function and you would call that from the other file as `MozuDropdown.mozuDropdown(...)` or similar.

Reply:
Agree this is not pure AMD module, but here I created JQuery plugin, I used AMD only to inject dependancy.